homeassistant/components/light/limitlessled.py

```diff
@@ -0,0 +1,112 @@
+"""
+homeassistant.components.light.limitlessled
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Support for LimitlessLED bulbs, also known as...
+
+EasyBulb
+AppLight
+AppLamp
+MiLight
+LEDme
+dekolight
+iLight
+
+light:
+  platform: limitlessled
+  host: 192.168.1.10
+  group_1_name: Living Room
+  group_2_name: Bedroom
+  group_3_name: Office
+  group_4_name: Kitchen
+
+"""
+import logging
+
+from homeassistant.helpers.entity import ToggleEntity
+from homeassistant.const import STATE_ON, STATE_OFF, DEVICE_DEFAULT_NAME
+from homeassistant.components.light import ATTR_BRIGHTNESS
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def setup_platform(hass, config, add_devices_callback, discovery_info=None):
+    """ Gets the LimitlessLED lights. """
+    try:
+        import ledcontroller
+    except ImportError:
+        _LOGGER.exception("Error while importing dependency ledcontroller.")
+        return
+
+    led = ledcontroller.LedController(config['host'])
+
+    lights = []
+    for i in range(1, 5):
+        if 'group_%d_name' % (i) in config:
+            lights.append(
+                LimitlessLED(
+                    led,
+                    i,
+                    config['group_%d_name' % (i)],
+                    STATE_OFF
+                )
+            )
+
+    add_devices_callback(lights)
+
+
+class LimitlessLED(ToggleEntity):
+    """ Represents a LimitlessLED light """
+
+    def __init__(self, led, group, name, state, brightness=180):
```

Comment:
Since you're always setting `state` to `STATE_OFF` and `brightness` to `180`, you might as well remove them from the constructor. Will clear the pylint warning.